platform/ui/src/components/measurementTable/MeasurementTable.styl

```diff
@@ -1,92 +1,116 @@
 .measurementTable
   background-color: var(--primary-background-color)
-  height: 100%;
-  width: 100%;
-  display: flex;
-  flex-direction: column;
+  height: 100%
+  width: 100%
+  display: flex
+  flex-direction: column
 
   .maxMeasurements
-    border-radius: 3px;
-    color: var(--primary-background-color);
-    font-size: 12px;
-    font-weight: 500;
-    height: 19px;
-    line-height: 17px;
-    margin-top: 22px;
-    margin-right: 15px;
-    padding: 2px 6px 0;
-    text-transform: uppercase;
-    transition: all 0.3s ease;
-    background-color: var(--default-color);
+    border-radius: 3px
+    color: var(--primary-background-color)
+    font-size: 12px
+    font-weight: 500
+    height: 19px
+    line-height: 17px
+    margin-top: 22px
+    margin-right: 15px
+    padding: 2px 6px 0
+    text-transform: uppercase
+    transition: all 0.3s ease
+    background-color: var(--default-color)
 
   .measurementTableHeader
-    padding-left: 45px;
+    padding-left: 45px
 
     .measurementTableHeaderItem
-      display: inline-block;
-      margin-top: 9px;
-      margin-left: 9px;
-      padding-left: 9px;
-      width: 90px;
-      font-size: 14px;
-      line-height: 20px;
-      border-left: 1px solid var(--text-secondary-color);
-      color: var(--text-primary-color);
+      display: inline-block
+      margin-top: 9px
+      margin-left: 9px
+      padding-left: 9px
+      width: 90px
+      font-size: 14px
+      line-height: 20px
+      border-left: 1px solid var(--text-secondary-color)
+      color: var(--text-primary-color)
 
       .timepointLabel
-        color: var(--text-secondary-color);
-        font-size: 12px;
+        color: var(--text-secondary-color)
+        font-size: 12px
 
     .warning-status
-      display: block;
-      position: absolute;
-      width: 50px;
-      height: 40px;
-      padding-left: 7px;
-      left: 15px;
-      top: 7px;
+      display: block
+      position: absolute
+      width: 50px
+      height: 40px
+      padding-left: 7px
+      left: 15px
+      top: 7px
 
       .warning-border
-        display: block;
-        border: 2px solid #e29e4a;
-        border-radius: 36px;
-        padding: 6px 7px;
-        width: 40px;
-        height: 40px;
+        display: block
+        border: 2px solid #e29e4a
+        border-radius: 36px
+        padding: 6px 7px
+        width: 40px
+        height: 40px
 
       svg
-        color: #e29e4a;
-        width: 22px;
-        height: 20px;
-        pointer-events: inherit;
+        color: #e29e4a
+        width: 22px
+        height: 20px
+        pointer-events: inherit
+
+  .measurementTableFooter
+    padding: 10px
+    color: #fff
+    text-align: center
+
+    .saveBtn
+      border: 1px solid var(--ui-border-color-active)
+      border-radius: 25px
+      background-color: var(--active-color)
+      color: var(--ui-gray-dark)
+      line-height: 25px
+      padding: 0 20px
+      outline: none
+      cursor: pointer
```

Comment:
We really need a good reusable button component, haha.
I feel like every component with a button has it's own styles.

Not a blocker.

Reply:
Yeah, I tried to get similar style as those rouded buttons in the toolbar section, but had to clone the style here for now.